sdk/state/payment.go

```diff
@@ -117,12 +118,12 @@ func (c *Channel) ProposePayment(amount int64) (CloseAgreement, error) {
 		return CloseAgreement{}, fmt.Errorf("cannot start a new payment while an unfinished one exists")
 	}
 
-	newBalance := int64(0)
-	if c.initiator {
-		newBalance = c.Balance() + amount
-	} else {
-		newBalance = c.Balance() - amount
+	finalPaymentAmount := amount
+	if !c.initiator {
+		finalPaymentAmount = amount * -1
```

Comment:
It looks like the payment amount is relative to the initiator, like the balance, but I think for ergonomics we should aim for it to be relative to the sender/receiver. We don't specify who the sender/receiver is explicitly, but implicitly it's always the proposer in our implementation. Relative to the proposer would make the value always positive since we only allow the proposal of payments to the confirmer.

Wdyt?

Note, if we go with `LastBalance`/`PreviousBalance` as the field to encode in the close agreement mentioned in my comment above, then I think this initiative relative approach makes sense.